lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -704,6 +706,7 @@ public static IGroupCommDriver GetInstanceOfGroupCommDriver(string driverId, str
                 .BindIntNamedParam<GroupCommConfigurationOptions.FanOut>(fanOut.ToString())
                 .BindIntNamedParam<GroupCommConfigurationOptions.NumberOfTasks>(numTasks.ToString())
                 .BindImplementation(GenericType<IConfigurationSerializer>.Class, GenericType<AvroConfigurationSerializer>.Class)
+                .BindImplementation(GenericType<ITcpPortProvider>.Class, GenericType<TcpPortProvider>.Class)
```

Comment:
This is redundant, right? `TcpPortProvider` is the default implementation for `ITcpPortProvider`, correct?